database/DevHome.Database/Services/RepositoryManagementDataAccessService.cs

```diff
@@ -30,20 +31,23 @@ public RepositoryManagementDataAccessService(DevHomeDatabaseContextFactory datab
     /// Makes a new Repository entity with the provided name and location then saves it
     /// to the database.
     /// </summary>
-    /// <param name="repositoryName">The name of the repository to add.</param>
-    /// <param name="cloneLocation">The local location the repository is cloned to.</param>
     /// <returns>The new repository.  Can return null if the database threw an exception.</returns>
-    public Repository? MakeRepository(string repositoryName, string cloneLocation, Uri repositoryUri)
+    public Repository MakeRepository(string repositoryName, string cloneLocation, string repositoryUri)
     {
         return MakeRepository(repositoryName, cloneLocation, string.Empty, repositoryUri);
     }
 
-    public Repository? MakeRepository(string repositoryName, string cloneLocation, string configurationFileLocationAndName, Uri repositoryUri)
+    public Repository MakeRepository(string repositoryName, string cloneLocation, string configurationFileLocationAndName, string repositoryUri)
+    {
+        return MakeRepository(repositoryName, cloneLocation, configurationFileLocationAndName, repositoryUri, Guid.Empty);
```

Comment:
Since `Repository.SourceControlClassId` is nullable, it seems clearer to pass `null` instead of `Guid.Empty`.

Reply:
Changed.